ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1165,7 +1170,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
+                                <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                                <Button Click="HUDAndVisuals_Reset" Width="22" Height="22" ToolTip="{DynamicResource ServerSettings_ResetSectionTooltip}" Margin="20,0,0,0">
```

Comment:
It would be better to use Commands instead of Click methods.  I discovered them after I had already created most of this UI.  See the RCON window for an example of how to use commands.  Commands have the advantage of being easily called from other places, they can be dynamically bound, and you can do nice things like have conditionals on them.